packages/engine/README.md

```diff
@@ -182,34 +224,35 @@ If one of the environment variables shown in the help page is passed, it will ov
 
 > **Warning** - Python and Rust runners are currently not supported. Within your simulation project, you should only see `.js` files within dependencies (for example, dependencies/@hash/age/src/behaviors/age.js). Files ending in `.rs` and `.py` will be ignored and the run will possibly fail in unclear ways. This also means that only an `init.json` or `init.js` is supported at the moment, not `init.py`.
 >
-> Currently, the easiest way of creating a project is by using the integrated IDE at [https://core.hash.ai][hCore] (hCore). In the absence of an in-depth description of expected project structure (which will be coming in the future), downloading a project from hCore is currently the easiest way to learn how one should be set out.
+> Currently, the easiest way of creating a project is by using the integrated IDE at [https://core.hash.ai][hcore] (hCore). In the absence of an in-depth description of expected project structure (which will be coming in the future), downloading a project from hCore is currently the easiest way to learn how one should be set out.
```

Comment:
Unrelated to this PR: do we have a ticket to actually outline the expected project structure? This seems pretty basic/central to "actually using HASH" and I think asking new users to download an existing simulation is especially poor DX. @hashintel/cloud

Reply:
I believed we did but a cursory search has not found it. I've added it to [one of our pseudo-epics](https://app.asana.com/0/0/1201646613812741/f)